### PR TITLE
Rename MinAffectedVersion in PackageIssue to AffectedVersion

### PR DESCRIPTION
**Pull request by nms2140**

### CODE REVIEW

**Comment by vtsao** [comment]
Want LGTM from @aysylu as well.

File: proto/v1/vulnerability.proto [hunk 1]
```diff
@@ -73,25 +73,35 @@ message VulnerabilityNote {
...
-    // The package this vulnerability was fixed in. It is possible for this to
-    // be different from the affected_package.
-    string fixed_package = 8;
-
-    // Required. The version of the package this vulnerability was fixed in.
-    // Setting this to VersionKind.MAXIMUM means no fix is yet available.
-    grafeas.v1.Version fixed_version = 9;
+    // Required. The version number at the start of an interval in which this vulnerability
+    // exists. A vulnerability can affect a package between version numbers
+    // that are disjoint sets of intervals (Example: [1.0.0-1.1.0], [2.4.6-2.4.8]
```
Comment (aysylu) [inline proto/v1/vulnerability.proto]: nit: let's use lower case 'e' here and ditto for the below

File: proto/v1/vulnerability.proto [hunk 2]
```diff
@@ -73,25 +73,35 @@ message VulnerabilityNote {
...
+    // and [4.5.6-4.6.8]).
```
Comment (aysylu) [inline proto/v1/vulnerability.proto]: Please add: "each of which will be represented in its own Detail"

Comment (aysylu) [reply proto/v1/vulnerability.proto]: ditto for the `affected_version_end` comment

Comment (aysylu) [inline proto/v1/vulnerability.proto]: Let's add a clarification that if a specific affection version is provided by a vulnerability database, `affected_version_start` and `affected_version_end` will be the same.

File: proto/v1/vulnerability.proto (hunk 1 above)
Comment (nms2140) [reply proto/v1/vulnerability.proto]: Addressed in Pull request #318

File: proto/v1/vulnerability.proto (hunk 2 above)
Comment (nms2140) [reply proto/v1/vulnerability.proto]: addressed in PR #318

Comment (nms2140) [reply proto/v1/vulnerability.proto]: Addressed in PR #318

**Pull request closed without merging**